src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffendersResourceTest.java

```diff
@@ -377,7 +378,7 @@ public void listAllOffenders() {
 
         assertThat(response.getHeaders().get("Page-Offset")).containsExactly("0");
         assertThat(response.getHeaders().get("Page-Limit")).containsExactly("100");
-        assertThat(response.getHeaders().get("Total-Records")).containsExactly("52");
+        assertThat(response.getHeaders().get("Total-Records")).containsExactly("53");
```

Comment:
:-)

Maybe we should write some SQL in the test to calculate the expected number

Reply:
true